docs/faq.md

```diff
@@ -35,13 +39,15 @@ Git Credential Manager Core (GCM Core; this project) aims to replace both GCM Wi
 
 ### Q: Does this mean GCM for Windows (.NET Framework-based) is deprecated?
 
-No. Git Credential Manager for Windows (GCM Windows) will continue to be supported until such a time that GCM Core is a complete replacement.
+Yes. Git Credential Manager for Windows (GCM Windows) is no longer receiving updates and fixes. All development effort has now been directed to GCM Core. GCM Core is available as an credential helper option in Git for Windows 2.28, and will be made the default helper in 2.29.
 
 ### Q: Does this mean the Java-based GCM for Mac/Linux is deprecated?
 
-Yes. Usage of Git Credential Manager for Mac and Linux (Java GCM) should be replaced with SSH keys. If you wish to take part in the public preview of GCM Core on macOS please feel free to install the latest preview release and give feedback! Otherwise, using SSH would be preferred on macOS and Linux to Java GCM.
+Yes. Usage of Git Credential Manager for Mac and Linux (Java GCM) should be replaced with GCM Core, or SSH keys. If you wish to install GCM Core on macOS or Linux please follow instructions [here](../README.md#download-and-install)!
```

Comment:
Spurious comma in `GCM Core, or SSH`. Missing comma in `macOS or Linux, please follow`.